### PR TITLE
SW-1449 Remove v1 accessions code from web-app codebase

### PR DESCRIPTION
**Pull request by karthikbtf**
- some refactor to keep shared constants
- removed cypress tests related to older flows
- did a bunch of manual testing
- if i missed anything let me know

### CODE REVIEW

File: src/api/seeds/accession.ts [hunk 1]
```diff
@@ -3,18 +3,6 @@ import { accessionEndpoint, checkInEndpoint, photoEndpoint } from '../types/acce
...
-
-export const postAccession = async (accession: any): Promise<number> => {
-  return 0;
-};
-
-export const putAccession = async (accessionId: number, accession: unknown): Promise<void> => {
-  return;
-};
-
 export const getPhotoEndpoint = (accessionId: number, photoFilename: string): string => {
```
Comment (constanzauanini) [inline src/api/seeds/accession.ts:6]: We can remove this endpoint too

Comment (karthikbtf) [reply src/api/seeds/accession.ts:6]: thanks, i thought we were using it for accessions2 as well.

**Review by joshbuchea — APPROVED** [review]
Changes are clean. Love PRs like this that remove so many lines of code.

**Review by constanzauanini — APPROVED** [review]
🎉

**Pull request merged**